modules/react-components/src/layouts/tab-page.tsx

```diff
@@ -76,3 +68,13 @@ export const TabPageLayout: FunctionComponent<PropsWithChildren<TabPageLayoutPro
         </PageLayout>
     );
 };
+
+/**
+ * Default props for the tab page layout.
+ */
+TabPageLayout.defaultProps = {
+    loadingStateOptions: {
+        count: 5,
+        imageType: "square"
```

Comment:
Shall we move this "square" to an constant or enum? WDYT?

Reply:
This is defined as a type in the interface.